Tasks/UseDotNetV1/utilities.ts

```diff
@@ -0,0 +1,9 @@
+import { chmodSync } from 'fs';
+
+export function getCurrentDir(): string {
```

Comment:
note, this is misleading since __dirname is dir of the file? not the process's current working directory

Reply:
Updated appropriately. This only really exists so that it can be easily mocked.